incubator/binding-grpc-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/grpc/kafka/streams/grpc/fetch/reliable.unary.rpc/client.rpt

```diff
@@ -34,7 +34,10 @@ write close
 
 read ${grpc:protobuf()
            .string(1, "Hello World")
-           .string(32767, "AQQABAIC")
+           .lastMessageId(32767)
+               .partition(0, 2)
+               .partition(1, 1)
+               .build()
```

Comment:
Let's move the `lastMessageId` concept to `GrpcKafkaFunctions`, and add the ability to have a binary field alongside a string field in the same protobuf message in `GrpcFunctions`.

Also, suggest renaming function to `messageId` instead.

```
read ${grpc:protobuf()
           .string(1, "Hello World")
           .bytes(32767, grpc_kafka:messageId()
               .partition(0, 2)
               .partition(1, 1)
               .build())
           .build()}
```

Note: the term `bytes` comes from protobuf v2 language guide, last row of table in [scalar types section](https://protobuf.dev/programming-guides/proto3/#scalar).